### PR TITLE
Additional fixes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-09 at 15 47 34|screen shot 2015-11-09 at 15 49 11|

### CODE REVIEW

**Comment by vxsx** [comment]
the current selection still looks bad

**Comment by lory87** [comment]
@vxsx screen shot 2015-11-09 at 16 24 59

File: djangocms_admin_style/sass/components/_forms.scss [hunk 1]
```diff
@@ -180,6 +183,10 @@ form {
             font-weight: normal;
         }
     }
+    .field-type label {
+       margin-left: 2px !important;
```
Comment (vxsx) [inline djangocms_admin_style/sass/components/_forms.scss]: space

**Comment by vxsx** [comment]
looks way better

**Comment by vxsx** [comment]
LGTM

**Pull request merged**